fix(kt-devnet): migrate kt-devnet to sepolia

iquidus opened the pull request:
Closes DAINT-729

Pins the teku image to version `25.7.1`
Bumps eigenda proxy version.
Migrates kt-devnet tests from holesky to sepolia

iquidus committed:
fix(kt-devnet): pin teku image to version `25.7.1`

samlaf reviewed (approved):
Did you copy this from some upstream PR? Could you have cherry-picked that instead?
Or at least mention in the PR description. Not super important but nice to have.

iquidus commented:
The change is part of upstream commit https://github.com/ethereum-optimism/optimism/commit/355fdcbb68310e70ffa6eb8a7635a4d9cea7935a, It includes a number of unrelated changes, that we will adopt with the next merge/rebase. This PR only updates the eigenda config to avoid conflicts.

iquidus committed:
fix(kt-devnet): replace holesky with sepolia

Replaces the existing holesky devnet config and tests with sepolia

iquidus committed:
fix(ci): remove `AUTOFIX=true` from devent workflows

iquidus committed:
test(kt-devnet): use holesky keys

iquidus committed:
fix(kt-devnet): bump proxy version and add network flag

Bumps the eigenda-proxy image version to 2.1.0.
Sets eigenda.v2.network flag to sepolia_testnet

iquidus committed:
drop proxy image down to v1.8.2

iquidus committed:
fix(kt-devnet): add support for v3 certs to failover tests

iquidus committed:
add disperser-rpc flags

iquidus committed:
add cert verifier addr flag

iquidus committed:
fix(kt-devnet): bump sepolia test timeout to 50m

iquidus committed:
fix(test): bump `TestBatcherFromLogs_Sepolia` timeout to 30m

iquidus committed:
ci: update signer keys to sepolia

iquidus committed:
remove v1 sepolia kt-devnet tests from ci

v1 tests are very slow and flaky on sepolia. Removing as we no
longer need them. Everyone should be using v2.
